### PR TITLE
Add help command for /poll htmlcreate"

### PR DESCRIPTION
**Pull request by spekss**

### CODE REVIEW

File: chat-plugins/poll.js [hunk 1]
```diff
@@ -312,6 +312,7 @@ exports.commands = {
 	pollhelp: ["/poll allows rooms to run their own polls. These polls are limited to one poll at a time per room.",
 				"Accepts the following commands:",
 				"/poll create [question], [option1], [option2], [...] - Creates a poll. Requires: % @ * # & ~",
+				"/poll htmlcreate [question], [option1], [option2], [...] - Creates a poll with HTML. Requires: % @ * # & ~",
```
Comment (panpawn) [inline chat-plugins/poll.js]: This line has trailing whitespace at the end of it, which is why Travis isn't passing

**Comment by spekss** [comment]
oh yes, I had not seen , thank you

**Comment by spekss** [comment]
finally ... it is late at night for me,
it's probably because I 'm tired I make mistakes too rinicules...

File: chat-plugins/poll.js [hunk 2]
```diff
@@ -312,6 +312,7 @@ exports.commands = {
 	pollhelp: ["/poll allows rooms to run their own polls. These polls are limited to one poll at a time per room.",
 				"Accepts the following commands:",
 				"/poll create [question], [option1], [option2], [...] - Creates a poll. Requires: % @ * # & ~",
+				"/poll htmlcreate [question, [option1], [option2], [...] - Creates a poll with HTML. Requires: % @ * # & ~",
```
Comment (SolarisFox) [inline chat-plugins/poll.js]: `[question` ]? :<

**Comment by Slayer95** [comment]
It's worth documenting that ``/poll htmlcreate`` may accept multiline input, and each line will be a parameter.

**Comment by SolarisFox** [comment]
It might be hard to fully explain htmlcreate in a concise manor, and many users will not care at all about it. If we're going to elaborate more on it, I think the further explanation should only be explained with `/help poll htmlcreate`

File: chat-plugins/poll.js (hunk 2 above)
Comment (Zarel) [inline chat-plugins/poll.js]: Also its requirements are higher iirc?

**Comment by spekss** [comment]
for /poll htmlcreate you think you need more explanation ?

**Comment by panpawn** [comment]
I propose this:
``"/poll htmlcreate [question], [option1], [option2], [...] - Creates a poll with HTML.  Each option can be on a separate line when the command is being sent. Requires: # & ~",``

Or something to that effect

**Comment by spekss** [comment]
I am going to replace it with what panpawn proposed, it seems better to me.

File: chat-plugins/poll.js [hunk 3]
```diff
@@ -312,6 +312,7 @@ exports.commands = {
 	pollhelp: ["/poll allows rooms to run their own polls. These polls are limited to one poll at a time per room.",
 				"Accepts the following commands:",
 				"/poll create [question], [option1], [option2], [...] - Creates a poll. Requires: % @ * # & ~",
+				"/poll htmlcreate [question], [option1], [option2], [...] - Creates a poll with HTML. Each option can be on a separate line when the command is being sent. Requires: # & ~",
```
Comment (Zarel) [inline chat-plugins/poll.js]: "Creates a poll with HTML." isn't exactly good English grammar here.

"Creates a poll, with HTML allowed in the question and options." is probably better.

Comment (spekss) [inline chat-plugins/poll.js]: okey i add this

File: chat-plugins/poll.js [hunk 4]
```diff
@@ -312,6 +312,7 @@ exports.commands = {
 	pollhelp: ["/poll allows rooms to run their own polls. These polls are limited to one poll at a time per room.",
 				"Accepts the following commands:",
 				"/poll create [question], [option1], [option2], [...] - Creates a poll. Requires: % @ * # & ~",
+				"/poll htmlcreate [question], [option1], [option2], [...] - Creates a poll, with HTML allowed in the question and options. Each option can be on a separate line when the command is being sent. Requires: # & ~",
```
Comment (Zarel) [inline chat-plugins/poll.js]: Remove "Each option can be on a separate line when the command is being sent."

It's pretty unclear and not really necessary to convey here.

**Comment by spekss** [comment]
done

**Pull request merged**